app/webpacker/src/javascripts/commodities.js

```diff
@@ -833,15 +847,7 @@
                   if (obj) {
                     $(element).parents('form:first').find('.js-commodity-picker-target').val(obj.id);
 
-                    $(document).trigger('tariff:chooseSearchQueryOption', [{
-                      params: {
-                        data: obj
-                      }
-                    }]);
-
-                    if (typeof($(element).data('nosubmit')) == 'undefined') {
```

Comment:
As I understand it, our preventing default behaviour is bypassing this. I'm going to revert it, though, just to avoid additional changes